### ISSUE TITLE
ObjectStatus: has incorrect attribute role="group"

### ISSUE THREAD

**Issue by ee92** [SAP/ui5-webcomponents-react#5980]
### Describe the bug

The element has role="group" which is not correct. 

sapui5 equivalent does not add the "group" role either:
https://sapui5.hana.ondemand.com/#/entity/sap.m.ObjectStatus/sample/sap.m.sample.ObjectStatus

### Reproduction steps

1. go to https://sap.github.io/ui5-webcomponents-react/?path=/docs/data-display-objectstatus--docs
2. using JAWS navigate to the ObjectStatus
3. notice the content is prefixed with "group start" and "group end"

### Expected Behaviour

Expect no aria role on the element

### UI5 Web Components for React Version

1.29.2

### UI5 Web Components Version

1.24.0

### Browser

Chrome

### Organization

SuccessFactors

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Hi @ee92 

according to Screen Reader support guidelines, inactive `ObjectStatus` should be announced as group. If you're working for SAP and have access to our internal systems, I can send you the resources (please contact me internally for this), otherwise you will unfortunately have to take my word for it.

**Comment by anuvenkatesh1** [comment]
role="group" is not suitable in this case because "group" is used to identify a set of related objects. (https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/group_role)
In our context, the object status is a "single object status" ie a single chip with a text on it. It doesn't even have an accompanying icon to group it along with, which is probably why the open UI5 sample (https://sapui5.hana.ondemand.com/#/entity/sap.m.ObjectStatus/sample/sap.m.sample.ObjectStatus) doesn't have this "grouping". 
Kindly reopen and reconsider the issue.

**Comment by Lukas742** [comment]
Hi @anuvenkatesh1 

since the role "group" is defined by our specifications, please open an internal accessibility incident and post the ticket ID here, so our accessibility experts can take a look. 

Also, even though I wouldn't recommend it, it's possible to change the role by passing the `role` prop:

```jsx
<ObjectStatus role="someRole">ObjectStatus</ObjectStatus>
```
